build/config/SignToolData.json

```diff
@@ -127,6 +127,7 @@
         "System.IO.FileSystem.DriveInfo.dll",
         "System.IO.FileSystem.Primitives.dll",
         "System.IO.Pipes.dll",
+        "System.Xml.ReaderWriter.dll",
```

Comment:
Not entirely sure. Probably something changed in .NET Core packages. The other Xml libs have a dependency on System.Xml.ReaderWriter, so I'm not that surprised that it's included by NuGet.